client/internal/bootstrap/config_test.go

```diff
@@ -7,192 +7,181 @@ import (
 	"encoding/json"
 	"os"
 	"path/filepath"
+	"testing"
 	"time"
 
 	"github.com/Azure/aks-secure-tls-bootstrap/client/internal/cloud"
 	"github.com/Azure/go-autorest/autorest/azure"
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
+	"github.com/stretchr/testify/assert"
 )
 
-var _ = Describe("config tests", func() {
-	Context("Config tests", func() {
-		Context("Validate", func() {
-			var cfg *Config
-
-			BeforeEach(func() {
-				cfg = &Config{
-					CloudProviderConfigPath: "path/to/azure.json",
-					APIServerFQDN:           "fqdn",
-					CustomClientID:          "clientId",
-					NextProto:               "alpn",
-					AADResource:             "appID",
-					ClusterCAFilePath:       "path",
-					KubeconfigPath:          "path",
-					CredFilePath:            "path",
-					Deadline:                time.Second,
+func TestConfig(t *testing.T) {
+	baseCfg := &Config{
```

Comment:
I'd prefer we define this within the test body within the for-loop below and avoid doing any copying